<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_21">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>241</width>
      <height>611</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Объекты</string>
     </property>
    </widget>
    <widget class="QListWidget" name="ObjListWidget"/>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ограничения</string>
     </property>
    </widget>
    <widget class="QListWidget" name="RestrListWidget"/>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выделенные объекты</string>
     </property>
    </widget>
    <widget class="QListWidget" name="SelectListWidget"/>
    <widget class="QPushButton" name="addSelectButton">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить
выделение</string>
     </property>
    </widget>
    <widget class="QLabel" name="Leftmessage">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Изменить/добавить  объект</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QCheckBox" name="CircleCheckBox">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Окружность</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="PointCheckBox">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string> Точка</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SegmentCheckBox">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Отрезок</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>x1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="x1LineEdit"/>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>x2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="x2LineEdit"/>
    </widget>
    <widget class="QSplitter" name="splitter_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>y1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="y1LineEdit"/>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>y2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="y2LineEdit"/>
    </widget>
    <widget class="QSplitter" name="splitter_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>rad</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="radLineEdit"/>
     <widget class="QPushButton" name="deleteObjButton">
      <property name="icon">
       <iconset>
        <normalon>:/delete.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="changeButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/addRestr.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="addOButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/parametrs.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="addPointButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/P.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="addCircleButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/cir.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="addSegmentButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/seg.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="SelectButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/select.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="fixButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/fix.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/search.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="decreaseButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/search.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>275</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="maximumSize">
      <size>
       <width>227</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить ограничение</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_16">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="P2PDISTCheckBox">
        <property name="font">
         <font>
          <family>Sans</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Расстояние
между точками</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="EQUALSCheckBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Равенство
отрезков</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="P2SDISTCheckBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Расстояние между
точкой и отрезком</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="ANGLECheckBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Угол между
отрезками</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="PARALCheckBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Параллельность</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="ORTHOCheckBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ортогональность</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID№1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id2LineEdit"/>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID№2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id1LineEdit"/>
    </widget>
    <widget class="QSplitter" name="splitter_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>restr</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="restrLineEdit"/>
     <widget class="QPushButton" name="addRestrPushButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/parametrs.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteRestrButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/delete.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>29</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>x:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>y:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="MyCanvas" name="myCanvas">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>36</y>
      <width>551</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="message">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>431</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyCanvas</class>
   <extends>QGraphicsView</extends>
   <header location="global">mycanvas.h</header>
   <slots>
    <signal>clickedx(double)</signal>
    <signal>clickedy(double)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
